hyperspy/io_plugins/digital_micrograph.py

```diff
@@ -821,6 +821,21 @@ def _parse_string(self, tag):
         else:
             return tag
 
+    def _get_EELS_exposure_time(self, tags):
+        # for GMS 2 and quantum/enfinium, the  "Integration time (s)" tag is
+        # only present for single spectrum acquisition;  for maps we need to
+        # compute exposure * number of frames
+        try:
+            if 'Integration_time_s' in tags.keys():
+                return float(tags["Integration time (s)"])
+            else:
+                frame_number = 1
+                if "Number_of_frames" in tags.keys():
+                    frame_number = float(tags["Number_of_frames"])
+                return float(tags["Exposure_s"]) * frame_number
+        except AttributeError:
```

Comment:
How is this going to raise an ``AttributeError``? Wouldn't ``if, elif, else`` be preferable in this case?